src/discord.ts

```diff
@@ -150,21 +154,37 @@ export class DiscordClient {
     }
   }
 
-  async onCommand(interaction: Interaction): Promise<void> {
-    if (!interaction.isCommand()) return;
-    const command = this.commands.get(interaction.commandName);
-    try {
-      if (command) await command.execute(interaction);
-      else interaction.reply(`Command not recognised. Something has gone wrong here.`);
-    } catch (error) {
-      console.log(error);
-      await (interaction.replied
-        ? interaction.followUp(
-            "OAF recovered from a crash trying to process that command. Please tell Scotch or Phill"
-          )
-        : interaction.reply(
-            "OAF recovered from a crash trying to process that command. Please tell Scotch or Phill"
-          ));
+  async onInteractionCreate(interaction: Interaction) {
+    if (interaction.isChatInputCommand()) {
+      const command = this.commands.get(interaction.commandName);
+
+      if (!command) {
+        console.error(`No command matching ${interaction.commandName} found`);
+        await interaction.reply(`Command not recognised. Something has gone wrong here.`);
+        return;
+      }
+
+      try {
+        await command.execute(interaction);
+      } catch (error) {
+        console.error(error);
+        await (interaction.replied ? interaction.followUp : interaction.reply)(
+          "OAF recovered from a crash trying to process that command. Please tell Scotch or Phill"
```

Comment:
swap this to just "tell Scotch" 

maybe "Please tell Scotch or report to #mafia-and-scripting" ?